spi/transfer-spi/src/main/java/org/eclipse/dataspaceconnector/spi/transfer/TransferProcessManager.java

```diff
@@ -39,6 +39,6 @@
     /**
      * Adds a single Command to the queue which is processed sequentially.
      */
-    void enqueueCommand(Command command);
+    void enqueueCommand(TransferProcessCommand command);
```

Comment:
Not sure this restriction is a good idea- what happens if someone just wants to add a command that operates on several TransferProcesses, like "count all in State XYZ"?